src/components/Search/SearchPageHeaderInput.tsx

```diff
@@ -143,46 +156,47 @@ function SearchPageHeaderInput({queryJSON, children}: SearchPageHeaderInputProps
         [autocompleteSubstitutions, originalInputQuery, queryJSON.policyID],
     );
 
-    const onListItemPress = (item: OptionData | SearchQueryItem) => {
-        if (!isSearchQueryItem(item)) {
-            return;
-        }
-
-        if (!item.searchQuery) {
-            return;
-        }
-
-        if (item.searchItemType === CONST.SEARCH.SEARCH_ROUTER_ITEM_TYPE.AUTOCOMPLETE_SUGGESTION && textInputValue) {
-            const trimmedUserSearchQuery = SearchAutocompleteUtils.getQueryWithoutAutocompletedPart(textInputValue);
-            onSearchQueryChange(`${trimmedUserSearchQuery}${SearchQueryUtils.sanitizeSearchValue(item.searchQuery)} `);
-
-            if (item.text && item.autocompleteID) {
-                const substitutions = {...autocompleteSubstitutions, [item.text]: item.autocompleteID};
-
-                setAutocompleteSubstitutions(substitutions);
+    const onListItemPress = useCallback(
+        (item: OptionData | SearchQueryItem) => {
+            if (isSearchQueryItem(item)) {
+                if (!item.searchQuery) {
+                    // Special case where empty item resets Search to default query
+                    Navigation.navigate(ROUTES.SEARCH_CENTRAL_PANE.getRoute({query: ''}));
```

Comment:
Can we not remove the navigation as we do not handle empty search queries?

Reply:
yes good catch. This should no longer be needed, will clean

Reply:
the if itself ` if (!item.searchQuery)` has to stay for TS, but the rest I addressed